#!/usr/bin/env python
PACKAGE = "first_project"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

odometry_enum = gen.enum([gen.const("wheel",     str_t, "wheel_odom", "Use wheel odometry for lidar remap"),
                          gen.const("gps",      str_t, "gps_odom", "Use gps odometry for lidar remap")],
                          "Reference odometry")

gen.add("reference_odometry", str_t, 0, "Parameter to set the refernce odometry", "wheel_odom", edit_method=odometry_enum)

exit(gen.generate(PACKAGE, "lidar_remap", "lidarRemap"))
